resources/install.rb

```diff
@@ -170,9 +170,7 @@
 
   case new_resource.source
   when 'distro'
-    log 'Using distro provided packages.' do
-      level :info
-    end
+    Chef::Log.debug 'Using distro provided packages.'
```

Comment:
I believe this should be `Chef::Log.info`

Reply:
Sure. I was and am a bit confused by what the OP wanted to have happen.  Changed it.